variable/technology/power-plant.yaml

```diff
@@ -0,0 +1,756 @@
+# List of variables related to power plants
+# Variables related to the production cost of a power plant
+# Note that:
+# linear   term cost   [€/GWh] = Linear Var Cost [Mcal/GWh] * Price|Primary Energy [€/Mcal] + Operation and Maintenance Variable Cost [€/MWh] * 1e-3
```

Comment:
I think that this is equivalent to the variables "Variable Cost|Electricity|Gas|OCGT:
   description: variable cost of a typical Open Cycle Gas Turbine power plant
   unit: €/MWh" and all others
except that I am not able to provide the formula how those were constructed , but the cost function that pla4EU is optimising when solving the unit commitment problem is 
Min Sum(all plants, all timesteps, all regions) Quadratic Cost* ActivePower^2 + variableCost * Active Power + FixedCost (when started) + StartUpCost (when starting)   
I only included variable Cost and StartUpCost in the nomenclature as for the others we do not have values in our public dataset.....

Reply:
Yeah, I think so.
On the other hand, openTEPES considers the parameters of this formula as an input.
We are not providing the Variable Cost as you do, because it is calculated in an internal way.
In my opinion, we can have all of these terms in the nomenclature.